### DISCUSSION TITLE
Cron jobs with a hoster

### DISCUSSION THREAD

**Discussion by zenotornado** [wavelog/wavelog#668] [Q&A] [answered]
Dear Community,

I have the situation that I run my Wavelog hosted and therefore have no possibility to create cronjobs via SSH. My hoster (Ionos) offers a manager to create cronjobs. Is there a way to use this for Wavelog as well?

![grafik](https://github.com/user-attachments/assets/f6d8d222-790f-430e-8bd8-ac4dab80205e)

**Comment by HB9HIL** [comment]
Hello @zenotornado 

What is the most often cronjob interval you can choose there?

**Reply by zenotornado** [reply]
It's daily... I would like to use it to sync the logs

**Reply by HB9HIL** [reply, marked as answer]
For Explaination: 

If you want to use the **implemented cronmanager** the "master cronjob" have to run every minute (* * * * *). 

If this is not available you still can add the cronjobs manually. A full list is available [here](https://github.com/wavelog/wavelog/wiki/Recommended-Cron-Jobs-and-Cronmanager#manual-cronjobs)

### How to in your case (Example): 

Name (optional): eQSL Sync

HTTP GET:  https://[Your-URL-To-Wavelog]/index.php/eqsl/sync

Status: On

Intervall: Whatever you like

**Reply by zenotornado** [reply]
Have set it now with the URL... let's await the job :-) Thank you

**Reply by 205er** [reply]
You can call the cron manager URL even from any external system.
In my case, I have a tiny linux machine at home which runs 24/7. There I setup a cron job which triggers the cronmanager at Wavelog on my hosted environment. Works like a charm.
